symphony/lib/toolkit/class.section.php

```diff
@@ -92,7 +92,16 @@ public function getDefaultSortingField()
      */
     public function getSortingField()
     {
-        $result = Symphony::Configuration()->get('section_' . $this->get('handle') . '_sortby', 'sorting');
+        $result = null;
+        
+        Symphony::ExtensionManager()->notifyMembers('getSortingField', '/publish/', array(
```

Comment:
Can you please add the required php doc for all `notifyMembers` calls ? It is required in order to generate the doc for the website.